task_test.go

```diff
@@ -2548,3 +2656,18 @@ func TestReference(t *testing.T) {
 		})
 	}
 }
+
+// overrideExperimentValue allows one to change the value of an experiment value for a set of tests,
+// with the value being restored to its previous state when tests complete.
+//
+// Typically experiments are controlled via TASK_X_ env vars, but we cannot use those in tests
+// because the experiment settings are parsed during experiments.init(), before any tests run.
+func overrideExperimentValue(t *testing.T, e *experiments.Experiment, enabled bool, val string) {
+	prev := *e
+	*e = experiments.Experiment{
+		Name:    prev.Name,
+		Enabled: enabled,
+		Value:   val,
+	}
+	t.Cleanup(func() { *e = prev })
+}
```

Comment:
@andreynering  Are you okay with this work-around?  I figure anything deeper would require changing the `experiments` package and it doesn't seem worth it to accommodate tests.

Reply:
I think this is good.
Imho, we do not need the `enabled` param, it'll be always true, as all experiments are disabled by default

Reply:
Good point.  I will adjust it.